pkg/collect/ceph.go

```diff
@@ -82,10 +90,25 @@ var CephCommands = []CephCommand{
 		DefaultTimeout: "30s", // include a default timeout because this command will hang if the RGW daemon isn't running/is unhealthy
 	},
 	{
-		ID:      "rbd-du", // the disk usage of each PVC
-		Command: []string{"rbd", "du"},
-		Args:    []string{"--pool=replicapool"},
-		Format:  "txt",
+		ID:             "rbd-du", // the disk usage of each PVC
+		Command:        []string{"rbd", "du"},
+		Args:           []string{"--pool=replicapool"},
+		Format:         "txt",
+		DefaultTimeout: "30s",
+	},
+	{
+		ID:             "osd-df",
+		Command:        []string{"ceph", "osd", "df"},
+		Args:           []string{"-f", "json-pretty"},
+		Format:         "json",
+		DefaultTimeout: "30s",
+	},
+	{
+		ID:             "df",
+		Command:        []string{"df"},
+		Args:           []string{"-h"},
+		Format:         "txt",
+		DefaultTimeout: "30s",
```

Comment:
will this give us anything useful, running inside the ceph pod?

Reply:
why not get the df output inside ceph instead of going back to host and run df again. the output is small as well.. 

```
[root@rook-ceph-operator-747c86774c-n7t28 /]# df
Filesystem     1K-blocks     Used Available Use% Mounted on
overlay        203070420 23785656 179268380  12% /
tmpfs              65536        0     65536   0% /dev
tmpfs           16436916        0  16436916   0% /sys/fs/cgroup
/dev/root      203070420 23785656 179268380  12% /etc/hosts
shm                65536        0     65536   0% /dev/shm
tmpfs           16436916       12  16436904   1% /run/secrets/kubernetes.io/serviceaccount
tmpfs           16436916        0  16436916   0% /proc/acpi
tmpfs           16436916        0  16436916   0% /proc/scsi
tmpfs           16436916        0  16436916   0% /sys/firmware
```